package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "elmi-to-json",
+  "version": "0.19.0",
```

Comment:
`elm-test` follows this convention for version numbers of having the `0.19` match the Elm version it works with, and the patch version just increments.  That has seemed to work fine for elm-test, though I use a more normal versioning scheme for elm-format, and also, I think elmi-to-json will only be used by advanced users, so the versioning scheme probably doesn't matter that much 🤷‍♀️

Reply:
sgtm